src/frequenz/sdk/microgrid/component/_component_data.py

```diff
@@ -128,6 +130,9 @@ class BatteryData(ComponentData):
             charging is possible.
         temperature_max: the maximum temperature of all the blocks in a battery, in
             Celcius (°C).
+        _relay_state: state of the battery relay.
+        _component_state: state of the battery.
+        _errors: list of errors in protobuf struct.
```

Comment:
It is OK for this PR, as there were already attributes documented like this, but in the future we need to move from documenting in `Attributes:` to documenting as docstrings where the attributes are defined, as the documentation tool we use renders the attributes much better if they are documented there.

Reply:
Done (last commit).